### PR TITLE
Fix compilation warnings

### LINKED ISSUE #32: Line 43:  Duplicate key 'calculatorInfo'  no-dupe-keys
two instances of the const `calculatorInfo` in the current program. This is producing the error, I don't believe it affects performance but it should be fixed.

### PR DESCRIPTION
**Pull request by vvv1559**
Fix #32 
Also fixed warning "Not used import"

### CODE REVIEW

**Comment by vvv1559** [comment]
Also fixed DOM error
`Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>...`

File: src/App.js [hunk 1]
```diff
@@ -36,14 +37,15 @@ import 'react-datepicker/dist/react-datepicker.css';
...
   constructor(props){
     super(props)
+    // set this to <false> if you don't want to have to read from an encoded URL, <true> instead
+    const readFromEncodedUrl = false;
     this.state = {
       numberOfDays: 0,
       result: "",
       resultDays: 0,
-      calculatorInfo: this.decodeToState(),
+      calculatorInfo: readFromEncodedUrl ? this.decodeToState() : excludedDates,
```
Comment (karismatic-megafauna) [inline src/App.js:46]: this is awesome, thank you for doing this!

**Comment by karismatic-megafauna** [comment]
fixes #35

File: src/App.js (hunk 1 above)
Comment (vvv1559) [reply src/App.js:46]: I am glad to help

**Pull request merged**